modules/ossm-architecture.adoc

```diff
@@ -7,15 +7,12 @@
+The *data plane* is a set of intelligent proxies deployed as sidecars. These proxies intercept and control all inbound and outbound network communication between microservices in the service mesh. 
 
-* *Envoy proxy* intercepts all inbound and outbound traffic for all services in the service mesh. Envoy is deployed as a sidecar to the relevant service in the same pod.
+The architectural units used by previous versions have been replaced by Istiod. In 2.0 the control plane components Mixer, Pilot, Citadel, Galley and the sidecar injector functionality have been combined into a single component, Istiod.
 
-The *control plane* manages and configures proxies to route traffic, and configures Mixers to enforce policies and collect telemetry.
+Although Mixer is no longer supported as a control plane component, Mixer policy and telemetry plugins are now supported through WebAssembly, or WASM, extensions in Istiod. For more information about how to use WebAssembly extensions, see *Extensions*. Mixer can be enabled for policy and telemetry if you need to integrate legacy Mixer plugins.
 
-* *Mixer* enforces access control and usage policies (such as authorization, rate limits, quotas, authentication, and request tracing) and collects telemetry data from the Envoy proxy and other services.
-* *Pilot* configures the proxies at runtime. Pilot provides service discovery for the Envoy sidecars, traffic management capabilities for intelligent routing (for example, A/B tests or canary deployments), and resiliency (timeouts, retries, and circuit breakers).
-* *Citadel* issues and rotates certificates. Citadel provides strong service-to-service and end-user authentication with built-in identity and credential management. You can use Citadel to upgrade unencrypted traffic in the service mesh. Operators can enforce policies based on service identity rather than on network controls using Citadel.
-* *Galley* ingests the service mesh configuration, then validates, processes, and distributes the configuration. Galley protects the other service mesh components from obtaining user configuration details from {product-title}.
+Security Discovery Service (SDS) is used to distribute certificates and keys to sidecars directly from Istiod. In {ProductName} version 1.1, secrets were generated by Citadel, which were used by the proxies to retrieve their client certificates and keys.
 
 {ProductName} also uses the *istio-operator* to manage the installation of the control plane. An _Operator_ is a piece of software that enables you to implement and automate common activities in your OpenShift cluster. It acts as a controller, allowing you to set or change the desired state of objects in your cluster.
```

Comment:
This is the first place we mention the control plane, and it's not bold formatting like the data plane mention.